api-doc.md

```diff
@@ -328,3 +328,33 @@
     - Headers : none
     - Message : Leave request has been sent successfully
 ```
+
+### 5. Send Update Leave Request mail
+
+```
+- Method : POST
+- Endpoint : /api/leave/update
+- Description : API for sending mail to Employee for their update leave request
```

Comment:
Replace with `API for sending Rejection/Approval mail of leave request to the employee`